service/service_test.go

```diff
@@ -276,12 +280,12 @@ func TestClose(t *testing.T) {
 
 			svcErrors := make(chan error, 1)
 			svcList := service.NewServiceList(initMock)
-			svc, err := service.Run(ctx, svcList, testBuildTime, testGitCommit, testVersion, svcErrors)
+			cfg, _ := config.Get()
```

Comment:
We should handle the error here, `t.FailNow()` ?

Reply:
This is a test , so I guess this is fine. Please let me know otherwise.

Reply:
Please change it, if it fails because it cant access config then it is good to know it was config error and not something else (error in later assertion could hide the real issue)

Reply:
Done.